learning/basics+/task1/solutions/PK-sum_of_natural.py

```diff
@@ -0,0 +1,17 @@
+"""
+All natural numbers (zero is considered a natural number for the purposes of the problem)
+divisible by 5 or 7 less than 20 are: [0, 5, 7, 10, 14, 15]. The sum of these numbers is: 51.
+Find the sum of all numbers divisible by 5 or 7 less than 100.
+Present the solution in the form of a function called calculate().
+In response, call the calculate() function and print the result to the console.
+Expected result:
+1580
+"""
+
+
+def calculate():
+    sum_of_numbers = sum([number for number in range(1,101) if number % 7 == 0 or number % 5 == 0 and number < 100])
```

Comment:
- Why you have `range(1,101)`, then restrict this range to >100 at the end? Just put `range(1,100)`
- Yeah abut that range - if you want range starting from 1, there is no need to put 1 as a first argument, `range(100)` will work well
```suggestion
    sum_of_numbers = sum([number for number in range(100) if number % 7 == 0 or number % 5 == 0])
```